luskan/src/main/java/org/opencadc/luskan/AdqlQueryImpl.java

```diff
@@ -105,7 +105,10 @@ protected void init() {
 
         super.navigatorList.add(new FunctionExpressionExtractor(
                 new PgFunctionNameConverter(), new ReferenceNavigator(), new FromItemNavigator()));
-        
+
+        // convert num_copies() -> cardinality(inventory.artifact.sitelocations)
+        super.navigatorList.add(new InventoryFunctionNavigator(new InventoryFunctionConverter()));
```

Comment:
I think you could move the `new InventoryFunctionConverter()` into the InventoryFunctionNavigator ctor (right in the super(...) like the others.

Reply:
Moved the creating the InventoryFunctionConverter into the InventoryFunctionNavigator constructor.